File: 0461-hamming-distance/0461-hamming-distance.java
class Solution {
    public int hammingDistance(int x, int y) {
        int xxoorr = x ^ y;
        int count = 0;
        
        while(xxoorr != 0)
        {
            if((1 & xxoorr) != 0)
                count++;
            xxoorr >>= 1;
        }
        return count;
    }
}

File: 0461-hamming-distance/0461-hamming-distance.swift
class Solution {
    func hammingDistance(_ x: Int, _ y: Int) -> Int {
        var diff = 0
        var xor = x ^ y
        
        while xor != 0 {
            if ((1 & xor) != 0) {
                diff += 1
            }
            xor = xor >> 1
        }
        return diff
    }
}